node/src/gql/mutation/me/update-mentor.ts

```diff
@@ -51,14 +51,12 @@ export const updateMentor = {
     context: { user: User }
   ): Promise<boolean> => {
     const mentorUpdate: MentorUpdateInput = args.mentor;
-    const mentor: Mentor = await MentorModel.findOne({ _id: mentorUpdate._id });
+    const mentor: Mentor = await MentorModel.findById(mentorUpdate._id);
     if (mentor && `${context.user._id}` !== `${mentor.user}`) {
       throw new Error('you do not have permission to update this mentor');
     }
-    const updated = await MentorModel.findOneAndUpdate(
-      {
-        _id: mentorUpdate._id,
-      },
+    const updated = await MentorModel.findByIdAndUpdate(
+      mentorUpdate._id,
```

Comment:
below, you shouldn't need to copy `mentorUpdate`, it's just:

```typescript
$set: mentorUpdate
```

...or I'm missing something?